Citation.cff

```diff
@@ -0,0 +1,13 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Laohaprapanon"
+  given-names: "Suriyan" 
+- family-names: "Sood"
+  given-names: "Gaurav"
+- family-names: "Naji"
+  given-names: "Bashar"
+title: "ethnicolr: Predict Race and Ethnicity From Name"
+version: 0.9.1
+date-released: 2022-06-17
+url: "https://github.com/appeler/naampy"
```

Comment:
The correct URL is:- https://github.com/appeler/ethnicolr

Reply:
fixed.